src/web/app/components/question-types/question-edit-answer-form/rank-options-question-edit-answer-form.component.html

```diff
@@ -6,7 +6,7 @@
       </label>
       <div class="col-sm-3">
         <select class="form-control" [ngModel]="responseDetails.answers[i]"
-                (ngModelChange)="triggerResponse(i, $event)">
+                (ngModelChange)="triggerResponse(i, $event)" [disabled]="isDisabled">
```

Comment:
The name should be `isEditable`?

Reply:
The component `question-edit-answer-form` which this extends from only has `isDisabled` which seems to be correct